clients/fossid-webapp/src/main/kotlin/EntityResponseBody.kt

```diff
@@ -23,6 +23,7 @@ class EntityResponseBody<T>(
     val operation: String? = null,
     val status: Int? = null,
     val message: String? = null,
+    val messageParameters: Array<String>? = null,
```

Comment:
Just a question: Do we evaluate this property somewhere or is it just mandatory to declare it because it is sent by FossID? If so, would it make sense to configure the JSON mapper to ignore unknown properties, so that we could keep our models minimalistic?

Reply:
Currently I don't know the content of this property, it is non documented.
I would like to have it inside to see what is coming: this could be useful.